test/e2e/dra/README.md

```diff
@@ -36,6 +36,177 @@ $ kind build node-image --image dra/node:latest $(pwd)
+  ---
+  apiVersion: cluster.x-k8s.io/v1beta1
+  kind: Cluster
+--- 313,325 ----
+    template:
+      spec:
+        joinConfiguration:
+!         nodeRegistration:
+!           kubeletExtraArgs:
+!             feature-gates: DynamicResourceAllocation=true,ContextualLogging=true
+!       initConfiguration:
+!         nodeRegistration:
+!           kubeletExtraArgs:
+!             feature-gates: DynamicResourceAllocation=true,ContextualLogging=true
+  ---
+  apiVersion: cluster.x-k8s.io/v1beta1
+  kind: Cluster
+***************
+*** 346,352 ****
+        - class: default-worker
+          name: md-0
+          replicas: 1
+-       machinePools:
+-       - class: default-worker
+-         name: mp-0
+-         replicas: 1
+--- 359,361 ----
+EOF
+
+# patch cluster-latest.yaml <cluster.patch
+patching file cluster-latest.yaml
+
+$ kubectl apply -f cluster-latest.yaml
+clusterclass.cluster.x-k8s.io/quick-start created
+dockerclustertemplate.infrastructure.cluster.x-k8s.io/quick-start-cluster created
+kubeadmcontrolplanetemplate.controlplane.cluster.x-k8s.io/quick-start-control-plane created
+dockermachinetemplate.infrastructure.cluster.x-k8s.io/quick-start-control-plane created
+dockermachinetemplate.infrastructure.cluster.x-k8s.io/quick-start-default-worker-machinetemplate created
+dockermachinepooltemplate.infrastructure.cluster.x-k8s.io/quick-start-default-worker-machinepooltemplate created
+kubeadmconfigtemplate.bootstrap.cluster.x-k8s.io/quick-start-default-worker-bootstraptemplate created
+cluster.cluster.x-k8s.io/capi-quickstart created
+
+# Some time later (minutes, not hours)...
+
+$ kubectl get cluster
+NAME              CLUSTERCLASS   PHASE         AGE   VERSION
+capi-quickstart   quick-start    Provisioned   47s   v1.29.0
+$ kubectl get machine
+NAME                                     CLUSTER           NODENAME                                 PROVIDERID                                          PHASE     AGE   VERSION
+capi-quickstart-h5bc8-wgppr              capi-quickstart   capi-quickstart-h5bc8-wgppr              docker:////capi-quickstart-h5bc8-wgppr              Running   89s   v1.29.0
+capi-quickstart-md-0-tzjwj-tb92p-wt4dc   capi-quickstart   capi-quickstart-md-0-tzjwj-tb92p-wt4dc   docker:////capi-quickstart-md-0-tzjwj-tb92p-wt4dc   Running   91s   v1.29.0
+
+$ clusterctl get kubeconfig capi-quickstart > capi-quickstart.kubeconfig
+
+$ KUBECONFIG=capi-quickstart.kubeconfig kubectl get nodes
+NAME                                     STATUS     ROLES           AGE     VERSION
+capi-quickstart-h5bc8-wgppr              NotReady   control-plane   2m26s   v1.29.0
+capi-quickstart-md-0-tzjwj-tb92p-wt4dc   NotReady   <none>          2m8s    v1.29.0
+
+# The control plane won’t be Ready until we install a CNI in the next step.
+
+$ KUBECONFIG=capi-quickstart.kubeconfig kubectl --kubeconfig=./capi-quickstart.kubeconfig \
+  apply -f https://raw.githubusercontent.com/projectcalico/calico/v3.26.1/manifests/calico.yaml
```

Comment:
you should use kindnetd is much more stable https://github.com/kubernetes-sigs/cluster-api/commit/d0c495a3bb0561f718dd804183700a9df6b2147e and consume less resources

Reply:
+1 Good catch!

Reply:
what can I say, it is my baby :)

Reply:
While we're here. Thx for maintaining kind & kindnet. It's very nice and stable foundation to build our own testing in CAPI on. Safes us so much time & effort.